recipes/hypothesis/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "3.4.2" %}
+{% set md5 = "b180f47bdbebdbbbfe4a6d2d7f95c640" %}
+{% set pkgname = "hypothesis" %}
+
+package:
+  name: {{ pkgname }}
+  version: {{ version }}
+
+source:
+  fn: simplejson-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ pkgname[0] }}/{{ pkgname }}/{{ pkgname }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - simplejson
+  run:
+    - python
+    - django
```

Comment:
It's in their [`extras_require`]( https://github.com/HypothesisWorks/hypothesis-python/blob/3.4.2/setup.py#L41-L45 ). Though I guess that raises the question as to whether we want to include all the `extras_require` or not.